### PR TITLE
Add company login alert to registration flow

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

**The login itself is not functional and will always fail at the moment, a concrete implementation of the login will follow.**

* Add company login alert to registration flow.
* The login screen now shows a Company Login button which manually shows the login alert.
* Use the login code auto-detection API provided by `wire-ios-sync-engine` to detect login codes 
in the clipboard and atomically show the alert on the registration flow screens and the landing page.

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -1131,6 +1131,15 @@
 "registration.no_history.restore_backup_failed.wrong_version.title" = "Incompatible backup";
 "registration.no_history.restore_backup_failed.wrong_version.message" = "This backup was created by a newer or outdated version of Wire and cannot be restored here.";
 
+// SSO – Company Login
+"login.sso.alert.title" = "Company Log In";
```
Comment (billypchan) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings]: We mix use "Login" & "Log In" in the app, should we unify them?

Comment (daehn) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: Yes, this is not final copy yet and will be changed once the final copy is ready.

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4738,6 +4742,7 @@
 				D5C34E8F203D75CA004A0986 /* Conversation Options */,
 				8FC852BC199245760008B66B /* UserProfile */,
 				874547691E9E54D500C206E5 /* SelfProfile */,
+				BFFA182920FDE23600F64377 /* SSO */,
```
[the remark this answers is not in the document]
Comment (daehn) [reply Wire-iOS.xcodeproj/project.pbxproj]: @vytis updated!

**Comment by daehn** [comment]
@vytis good point, will adjust the naming globally now.

**Pull request merged**